src/PAModel/Schemas/ResourcesJson.cs

```diff
@@ -36,17 +47,16 @@ internal class ResourceJson
         public ResourceKind ResourceKind { get; set; }
         public string Path { get; set; }
         public ContentKind Content { get; set; }
+        public string Schema { get; set; }
```

Comment:
Does this need to be a string, or can it be the enum? System.Text.Json should be able to handle serializing/deserializing the enum written out

Reply:
Can't use enum type for the the character used for unknown types '?'